be/src/runtime/string_value.h

```diff
@@ -60,10 +105,36 @@ struct StringValue {
     // this < other: -1
     // this == other: 0
     // this > other: 1
-    int compare(const StringValue& other) const;
+    int compare(const StringValue& other) const {
```

Comment:
small function add key word `inline`，the same for following `eq`

Reply:
done